src/Swashbuckle.AspNetCore.Swagger/SwaggerMiddleware.cs

```diff
@@ -4,6 +4,9 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Routing;
+#if !NETSTANDARD
```

Comment:
I think it would be a bit more readable to use something like `#if NET5_OR_GREATER` (or whatever the appropriate one is) for these, rather than _"not .NET Standard"_. Otherwise I had to go to check the project file to see if these also applied to any `net4x` TFMs (which it turns out we don't target).

Reply:
I think it's correct to use `#if !NETSTANDARD` as it mirrors the condition on the `$(TargetFramework)` in the csproj files. Also, there are many other places where `#if (!NETSTANDARD2_0)` is used.

Anyway, once #2798 is implemented this will become moot as all conditional compilation can be removed.

Reply:
Happy to leave it as is, though I think the other one is more readable.

Regarding #2798, that might be a while yet.